setup.py

```diff
@@ -39,7 +39,7 @@
     install_requires=[
         "numpy",
         "PyYAML",
-        "torch-mlir",
+        "torch-mlir==20230802.121",
```

Comment:
wait wait why

Reply:
Just testing, it wouldn't pull the newest and was instead pulling from earlier today.

Reply:
We don't need the pin when merging

Reply:
Resolving as this pin was removed.